Library/Homebrew/test/test_build_options.rb

```diff
@@ -20,6 +20,7 @@ def test_with_without
     assert @build.with?("bar")
     assert @build.with?("baz")
     assert @build.without?("qux")
+    assert @build.without?("dogs")
```

Comment:
Ah I see, Apologies! Because the 7 and 8 lines in this file were unmodified I thought the change here was related to the one in the other file.